### ISSUE TITLE
SyntaxError: Unexpected token only with PRETTY_PRINT

### ISSUE THREAD

**Issue by mikob** [ampproject/rollup-plugin-closure-compiler#206]
## What's the issue?

Setting formatting to PRETTY_PRINT causes syntax error.

## How do we reproduce the issue?
@ampproject/rollup-plugin-closure-compiler@0.10.0
rollup@1.19.4
Node v12.4.0
Linux Mint 19.1 64bit

```rollup.config.js
import compiler from '@ampproject/rollup-plugin-closure-compiler';

export default {
	input: `/tmp/extension-util.js`,
	plugins: [
		compiler({
			languageOut: 'ECMASCRIPT_2019',
			formatting: 'PRETTY_PRINT',
		}),
	],
	treeshake: false,
	output: {
		file: `/tmp/output.js`,
		format: 'esm',
	}
}
```

input file: https://gist.github.com/mikob/acd7906db53ed5cf9e4178c853b951dc

## Additional context
The actual issue seems to have nothing to do with the line in question:
```
      return (frame.functionName || '').indexOf('StackGenerator$$') === -1;
```
because when I move code around, it will happen on different seemingly arbitrary lines. Perhaps a memory issue?

```
src/modules/extension-util.ts → package/dist/modules/extension-util.js...
(node:27095) UnhandledPromiseRejectionWarning: SyntaxError: Unexpected token (1945:15)
    at _class.pp$4.raise (/home/mikob/workspace/lipsurf/chrome-extension/node_modules/acorn/dist/acorn.js:2828:15)
    at _class.pp.unexpected (/home/mikob/workspace/lipsurf/chrome-extension/node_modules/acorn/dist/acorn.js:689:10)
    at _class.pp$3.parseExprAtom (/home/mikob/workspace/lipsurf/chrome-extension/node_modules/acorn/dist/acorn.js:2272:12)
    at _class.parseExprAtom (/home/mikob/workspace/lipsurf/chrome-extension/node_modules/acorn-dynamic-import/lib/index.js:75:117)
    at _class.pp$3.parseExprSubscripts (/home/mikob/workspace/lipsurf/chrome-extension/node_modules/acorn/dist/acorn.js:2091:21)
    at _class.pp$3.parseMaybeUnary (/home/mikob/workspace/lipsurf/chrome-extension/node_modules/acorn/dist/acorn.js:2068:19)
    at _class.pp$3.parseExprOps (/home/mikob/workspace/lipsurf/chrome-extension/node_modules/acorn/dist/acorn.js:2012:21)
    at _class.pp$3.parseMaybeConditional (/home/mikob/workspace/lipsurf/chrome-extension/node_modules/acorn/dist/acorn.js:1995:21)
    at _class.pp$3.parseMaybeAssign (/home/mikob/workspace/lipsurf/chrome-extension/node_modules/acorn/dist/acorn.js:1970:21)
    at _class.pp$1.parseExport (/home/mikob/workspace/lipsurf/chrome-extension/node_modules/acorn/dist/acorn.js:1433:33)
(node:27095) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:27095) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by kristoferbaxter** [comment]
Took a look with the debugger enabled... this is going to take awhile to figure out.

Here's the output post closure compilation, and acorn is unable to parse.

https://gist.github.com/kristoferbaxter/88105027b3878eaabc4b752639ebd730

**Comment by kristoferbaxter** [comment]
If you're interested in debugging, you can enable the debugger in `debug.ts`, and locally link the installation of `@ampproject/rollup-plugin-closure-compiler`.

**Comment by kristoferbaxter** [comment]
Found the root cause here, several transforms assume they are running on minified code post compilation.

We should enable testing for all conditions with pretty printing to fix and keep regressions from occurring.

**Comment by kristoferbaxter** [comment]
Resolved with 847bca7. Will release with `0.20.0`.
